### ISSUE TITLE
Transparent apple emoji aren't transparent

### ISSUE THREAD

**Issue by Gankro** [servo/webrender#1780]
```
data:text/html,<html><head><meta charset="utf8"></head><body><span style="color: blue">%F0%9F%8E%89</span><span style="color: rgba(0,0,0,0.7)">%F0%9F%8E%89</span><span style="color: rgba(0,0,255,0.4)">%F0%9F%8E%89</span><span style="color: transparent">%F0%9F%8E%89</span>
```

```html
<html>
<head><meta charset="utf8"></head>
<body>
    <span style="color: blue">🎉</span>
    <span style="color: rgba(0,0,0,0.7)">🎉</span>
    <span style="color: rgba(0,0,255,0.4)">🎉</span>
    <span style="color: transparent">🎉</span>
</body>
```

Chrome:
![screen shot 2017-09-29 at 3 00 01 pm](https://user-images.githubusercontent.com/1136864/31031594-069e43b2-a527-11e7-829a-5ff8268e1522.png)

Webrendest Firefox:
![screen shot 2017-09-29 at 3 00 22 pm](https://user-images.githubusercontent.com/1136864/31031601-06fd1838-a527-11e7-9a65-e76e8244b6f1.png)

Partially this is my fault, because I missed that firefox treats this the same as synthetic bold, doing the transparency as a compositing affect (and so wr was being sent opaque colors). However this should be fixed by https://bugzilla.mozilla.org/show_bug.cgi?id=1404370, and yet with that patch we still get opaque.

**Comment by mstange** [comment]
After #1816 we now seem to be applying the alpha to the emojis, but we're also applying the color components of the color again. 
<img width="75" alt="screen shot 2017-10-10 at 6 37 06 pm" src="https://user-images.githubusercontent.com/961291/31414048-0b61a184-adea-11e7-8a66-8782c8ce384f.png">

**Comment by mstange** [comment]
Fixed by #1945.
